docs/manage/configuration.md

```diff
@@ -0,0 +1,298 @@
+---
+title: "Configure a Robot"
+linkTitle: "Configure a Robot"
+weight: 25
+type: "docs"
+description: "Before you can program a robot, you must configure its components and services as well as any modules, remotes, processes and frames."
+tags: ["manage", "components"]
+---
+
+Before you can program a robot, you must configure it.
+
+A robot's configuration tells the code running the robot what *resources* (hardware *components* and software *services*) it has access to, as well as any relevant parameters for those resources.
+
+To configure your robot, go to the [Viam app](https://app.viam.com), [create a new robot](/installation/install/) and follow the steps on the **SETUP** page.

```

Comment:
```suggestion
To configure your robot, go to the [Viam app](https://app.viam.com), [create a new robot](/installation/) and follow the steps on the **SETUP** page.

```

Reply:
This was the link you suggested in the alternate flow thing so I applied it, but I’m actually wondering if it doesn’t make sense because “create a robot” =/= install. Maybe this sentence should actually be 
```suggestion
To start configuring, go to the [Viam app](https://app.viam.com), create a new robot and follow the steps on your new robot’s **SETUP** tab.
```

Reply:
Sure